docs/src/app/components/pages/components/date-picker.jsx

```diff
@@ -30,6 +30,12 @@ export default class DatePickerPage extends React.Component {
         name: 'Props',
         infoArray: [
           {
+            name: 'container',
+            type: 'one of: dialog, container',
+            header: 'default: dialog',
+            desc: 'The date pickers container type',
```

Comment:
@mbrookes @oliviertassinari I've made the changes in https://github.com/callemall/material-ui/pull/2152. 

Please let me know if you have any other suggestions.